Client/core/Graphics/CGraphics.cpp

```diff
@@ -483,43 +485,42 @@ void CGraphics::EndDrawBatch()
 //
 void CGraphics::CheckModes(EDrawModeType newDrawMode, EBlendModeType newBlendMode)
 {
-    bool bDrawModeChanging = (m_CurDrawMode != newDrawMode);
-    bool bBlendModeChanging = (m_CurBlendMode != newBlendMode);
-    // Draw mode changing?
-    if (bDrawModeChanging || bBlendModeChanging)
```

Comment:
I think this old code is clearer and makes it more obvious what's happening

Reply:
Yeah, you're right. I'll change it back.